Real Estate Website – Technigo Bootcamp Project 2025

KausarShangareeva opened the pull request:
Hi there! 👋
I’m currently building DMian, a real estate website focused on the Emirates. The project showcases property listings in a clean, modern design, fully responsive across devices.

> Check out the live website here: [DMian](https://dian-eng.netlify.app/)

Key Features:

- Clean, green-themed design for a fresh look
- Fully responsive layout using Flexbox
- Interactive elements with JavaScript for a better user experience
- Structured with HTML & CSS for simplicity and maintainability

This project helped me practice responsive layouts, interactivity, and combining HTML, CSS, and JavaScript to create a functional and visually appealing website. 🌟

KausarShangareeva committed:
Finally all

HIPPIEKICK reviewed (changes requested):
Good job with this business site, Kausyar, looks really nice!
However, it seems like you're missing some requirements:
- Responsive hamburger menu that's visible only on mobile devices
- You have divs/spans that _look_ like a form, but I can't see any actual form element which is one of the requirements this week

To add to this, I'm having a hard time checking some of the requirements due to the large amount of files in your repository. Would you mind removing the node_modules (are you using them?) and/or creating a .gitignore file where you write `node_modules` to make sure they are excluded on GitHub.

Also, there's a little bug when I click the AUD button, then some text appears on top of the existing text. So please also fix this.
<img width="1168" height="461" alt="Image" src="https://github.com/user-attachments/assets/bb7217f9-ecfe-430a-80f6-d3613dc917ac" />

mikaelasturk commented on css/style.css:
```diff
@@ -0,0 +1,1464 @@
+/* ------------------------------------------------------
+  | THEME VARIABLES: базовые (light) ☀️
+  ------------------------------------------------------ */
+:root {
```
Good practice to use root!

mikaelasturk commented on css/style.css:
```diff
@@ -0,0 +1,1464 @@
+  height: 1000rem;
+}
+
+body {
+  opacity: 0;
+  transition: opacity 0.3s ease-in-out;
+}
+
+body.loaded {
+  opacity: 1;
+}
```
This was a really cool detail to add!

mikaelasturk commented on css/style.css:
```diff
@@ -0,0 +1,1464 @@
+/* -------------------------------------
+  |  5️⃣ Extra small phone: ≤ 320px 5️⃣ |
+  ------------------------------------ */
+/* 
```
Did you forget to put these in the right place? I love the clear, big comments!

mikaelasturk commented on js/script.js:
```diff
@@ -0,0 +1,207 @@
+  }
+});
+
+const toggle = document.getElementById("language-toggle");
+toggle.addEventListener("click", () => {
+  localStorage.setItem("scrollPos", window.scrollY);
+
+  const currentLang = window.location.pathname.includes("index.en.html")
+    ? "en"
+    : "ru";
+  if (currentLang === "ru") {
+    window.location.href = "index.en.html";
+  } else {
+    window.location.href = "index.html";
+  }
+});
```
Really good feature, easy to read. Relevant to the project, so that you can work in your preferred language.

mikaelasturk commented on index.en.html:
```diff
@@ -0,0 +1,1796 @@
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>ДИАН - компания по недвижимости</title>
```
Don't forget to add an english title here :)

mikaelasturk reviewed (commented):
I really like the clear and detailed comments, especially in the HTML file! It makes the code easy to follow and understand.

It would be great to see a few more comments in the CSS as well, and preferably in English to keep everything consistent. ;)

I also noticed that the hamburger menu is missing from the project, which is part of the requirements. :) 

Overall, great work!! The website feels advanced and thoughtfully built!

KausarShangareeva committed:
Final changes

KausarShangareeva committed:
Add responsive search form with property, district, and price filters

KausarShangareeva committed:
Update README to remove price filter and favorites

Removed price filter and favorites system from features list. Updated how it works section to reflect changes.

KausarShangareeva committed:
Update README.md

KausarShangareeva commented:
> Good job with this business site, Kausyar, looks really nice! However, it seems like you're missing some requirements:
> 
> * Responsive hamburger menu that's visible only on mobile devices
> * You have divs/spans that _look_ like a form, but I can't see any actual form element which is one of the requirements this week
> 
> To add to this, I'm having a hard time checking some of the requirements due to the large amount of files in your repository. Would you mind removing the node_modules (are you using them?) and/or creating a .gitignore file where you write `node_modules` to make sure they are excluded on GitHub.
> 
> Also, there's a little bug when I click the AUD button, then some text appears on top of the existing text. So please also fix this. <img alt="Image" width="1168" height="461" src="https://private-user-images.githubusercontent.com/51529201/499386684-bb7217f9-ecfe-430a-80f6-d3613dc917ac.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjMzMjAyMTAsIm5iZiI6MTc2MzMxOTkxMCwicGF0aCI6Ii81MTUyOTIwMS80OTkzODY2ODQtYmI3MjE3ZjktZWNmZS00MzBhLTgwZjYtZDM2MTNkYzkxN2FjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTExMTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUxMTE2VDE5MDUxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThhMDhhNjRmYmI2OTY4MjIxZjJjY2I3MWI2Yzg3ZDZjY2Y5ZTE0MGQ1Y2QyYWQ2OGIwZjc1MTk0ZGY2M2YzZWQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.pcaKIKyFbjLvz71XrmhTLYcayIlzJnNKLLY9zVtWwU8">

Im done ✅😀

HIPPIEKICK commented:
> Good job with this business site, Kausyar, looks really nice! However, it seems like you're missing some requirements:
> 
> * Responsive hamburger menu that's visible only on mobile devices
> * You have divs/spans that _look_ like a form, but I can't see any actual form element which is one of the requirements this week
> 
> To add to this, I'm having a hard time checking some of the requirements due to the large amount of files in your repository. Would you mind removing the node_modules (are you using them?) and/or creating a .gitignore file where you write `node_modules` to make sure they are excluded on GitHub.
> 
> Also, there's a little bug when I click the AUD button, then some text appears on top of the existing text. So please also fix this. <img alt="Image" width="1168" height="461" src="https://private-user-images.githubusercontent.com/51529201/499386684-bb7217f9-ecfe-430a-80f6-d3613dc917ac.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjQxNDc3NTAsIm5iZiI6MTc2NDE0NzQ1MCwicGF0aCI6Ii81MTUyOTIwMS80OTkzODY2ODQtYmI3MjE3ZjktZWNmZS00MzBhLTgwZjYtZDM2MTNkYzkxN2FjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTExMjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUxMTI2VDA4NTczMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBmZDg3MzQzYzIwN2I0NDA0MzczZDZjOTMxNDllNjI1MGQzZjFmZTQzNmNlNDgzZTY4N2Y0YTQwOTY0ZTIzZTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.6RrhgUQl7cN1nAzmIVrgOF_jbkHppRsWDvaF7901EEA">

Can't you see my comment from October 9, @KausarShangareeva ?

HIPPIEKICK commented:
- Still the same bug and still no hamburger menu:
<img width="412" height="706" alt="Screenshot 2025-11-27 at 10 15 04" src="https://github.com/user-attachments/assets/53501a0b-8826-4ccb-8dc1-7bf403f00f51" />

- Node modules still there, please remove them
- Form is present ✅

KausarShangareeva committed:
Update README.md
